File: StarWars_Python/PersoGentil.py
from Personnage import Personnage



class PersoGentil(Personnage):

    def __init__(self, strength=None):
        super().__init__()
        self.strength = strength

    def __str__(self):
        return f"PersoGentil{super().__str__()} [Force: {self.strength}]"

File: OneDrive/Bureau/SLAM/Mission7/Starwars/StarWars_Java/src/main/java/PersoGentil.java
package starwars;

public class PersoGentil extends Personnage{
	private Boolean Force;

	
	public PersoGentil(Boolean Force) {
		super();
        this.Force = Force;
    }
	public Boolean getForce() {
		return Force;
	}
	public void setForce(Boolean force) {
		Force = force;
	}
	public String toString() {
		return "PersoGentil"+super.toString()+" [Force=" + Force + "]";
	}
}